br/pkg/task/backup.go

```diff
@@ -86,6 +86,9 @@ func DefineBackupFlags(flags *pflag.FlagSet) {
 		"backup sst file compression algorithm, value can be one of 'lz4|zstd|snappy'")
 	flags.Int32(flagCompressionLevel, 0, "compression level used for sst file compression")
 
+	flags.Uint32(flagConcurrency, 4, "The size of thread pool on br that executes the task," +
+		"this value should increased when backup large amount of table + index")
```

Comment:
if you mention this you need to explain what's the criteria of increasing the concurrency and what's the value (which then leads to why it can't be automatically determined etc)

Reply:
@3pointer i mean explaining *why* you need to increase, not *what is the meaning of a task*. for instance if i'm going to backup 10000 tables does "increasing" means i should set `-c 10000`? 

as this is making things complicated and makes the `--help` page verbose i think it's better to eliminate the suggestion and put the detail in the docs.

```suggestion
```

Reply:
update the comment. PTAL~

Reply:
but, only output `The size of thread pool on BR that executes tasks` will still make user confused. I agree we should mention the details in document. @IANTHEREAL WDYT?

Reply:
I have never seen other products allow to adjust the concurrency in restore. 
We do not need to control the impact when restoring. Can it be made into automatic adjustment?